src/redux/actions/cellSets/loadCellSets.js

```diff
@@ -30,6 +32,7 @@ const loadCellSets = (experimentId) => async (dispatch, getState) => {
       payload: {
         experimentId,
         data: json.cellSets,
+        order: sampleIds,
```

Comment:
We could access this value inside the CELL_SETS_LOADED reducer from the current state, right? Instead of passing it as parameter.

Reply:
Yes, we can. Right, I'll move it into the reducer.